<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>奥运会统计</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>375</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../image/medal.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>按国家编号查询:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="country_no">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_country">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>按项目编号查询:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="event_no">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_event">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>数据管理</string>
    </property>
    <addaction name="action_input"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>多方法排序</string>
    </property>
    <addaction name="action_no"/>
    <addaction name="action_score"/>
    <addaction name="action_man"/>
    <addaction name="action_woman"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_input">
   <property name="text">
    <string>重输数据</string>
   </property>
  </action>
  <action name="action_no">
   <property name="text">
    <string>编号排序</string>
   </property>
  </action>
  <action name="action_score">
   <property name="text">
    <string>总分排序</string>
   </property>
  </action>
  <action name="action_man">
   <property name="text">
    <string>男团排序</string>
   </property>
  </action>
  <action name="action_woman">
   <property name="text">
    <string>女团排序</string>
   </property>
  </action>
  <action name="action5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
